staging/publishing/rules.yaml

```diff
@@ -290,6 +290,13 @@ rules:
       branch: release-1.25
       dir: staging/src/k8s.io/component-helpers
   library: true
+- destination: kms
+  library: true
+  branches:
+  - source:
+      branch: master
```

Comment:
```suggestion
      branch: main
```

Reply:
@ameukam `kubernetes/kubernetes` default branch is still master, so if we change this to `main` would it work?

Reply:
@aramase did the suggestion on the wrong place (blame the Github UI).